package NRl;

/* Numbers must be consecutive, because they are used to define mapping from enum to string.
 * Locale codes according to http://en.wikipedia.org/wiki/ISO_3166-1
 */
enum ERelevLocale {
    RL_UNIVERSE = 0;   /* "universe" */
    RL_RU      = 1;   /* "ru" */
    RL_UA      = 2;   /* "ua" */
    RL_BY      = 3;   /* "by" */
    RL_KZ      = 4;   /* "kz" */
    RL_TR      = 5;   /* "tr" */
    RL_DE      = 6;   /* "de" */
    RL_IT      = 7;   /* "it" */
    RL_PL      = 8;   /* "pl" */
    RL_LT      = 9;   /* "lt" */
    RL_LV      = 10;  /* "lv" */
    RL_EE      = 11;  /* "ee" */
    RL_MD      = 12;  /* "md" */
    RL_RO      = 13;  /* "ro" */
    RL_US      = 14;  /* "us" */
    RL_GE      = 15;  /* "ge" */
    RL_UZ      = 16;  /* "uz" */
    RL_IL      = 17;  /* "il" */
    RL_FR      = 18;  /* "fr" */
    RL_CA      = 19;  /* "ca" */
    RL_GB      = 20;  /* "gb" */
    RL_HU      = 21;  /* "hu" */
    RL_AZ      = 22;  /* "az" */
    RL_BG      = 23;  /* "bg" */
    RL_IN      = 24;  /* "in" */
    RL_CZ      = 25;  /* "cz" */
    RL_SA      = 26;  /* "sa" */
    RL_VN      = 27;  /* "vn" */
    RL_DK      = 28;  /* "dk" */
    RL_ID      = 29;  /* "id" */
    RL_IE      = 30;  /* "ie" */
    RL_ES      = 31;  /* "es" */
    RL_CN      = 32;  /* "cn" */
    RL_KR      = 33;  /* "kr" */
    RL_NL      = 34;  /* "nl" */
    RL_NO      = 35;  /* "no" */
    RL_PT      = 36;  /* "pt" */
    RL_TH      = 37;  /* "th" */
    RL_PH      = 38;  /* "ph" */
    RL_JP      = 39;  /* "jp" */
    RL_AU      = 40;  /* "au" */
    RL_BE      = 41;  /* "be" */
    RL_BR      = 42;  /* "br" */
    RL_EG      = 43;  /* "eg" */
    RL_MX      = 44;  /* "mx" */
    RL_NZ      = 45;  /* "nz" */
    RL_PR      = 46;  /* "pr" */
    RL_CH      = 47;  /* "ch" */
    RL_WORLD   = 48;  /* "world" */
    RL_XUSSR   = 49;  /* "xussr" */
    RL_XCOM    = 50;  /* "xcom" */
    RL_SPOK    = 51;  /* "spok" */
    RL_MY      = 52;  /* "my" */
    RL_AM      = 53;  /* "am" */
    RL_KG      = 54;  /* "kg" */
    RL_TJ      = 55;  /* "tj" */
    RL_TM      = 56;  /* "tm" */
    RL_RS      = 57;  /* "rs" */
    RL_CI      = 58;  /* "ci" */
    RL_FI      = 59;  /* "fi" */
    RL_EU      = 60;  /* "eu" */
    RL_AT      = 61;  /* "at" */
    RL_HR      = 62;  /* "hr" */
    RL_CY      = 63;  /* "cy" */
    RL_GR      = 64;  /* "gr" */
    RL_IS      = 65;  /* "is" */
    RL_LI      = 66;  /* "li" */
    RL_LU      = 67;  /* "lu" */
    RL_MT      = 68;  /* "mt" */
    RL_SK      = 69;  /* "sk" */
    RL_SI      = 70;  /* "si" */
    RL_SE      = 71;  /* "se" */
    RL_CM      = 72;  /* "cm" */
    RL_SN      = 73;  /* "sn" */
    RL_AO      = 74;  /* "ao" */
};
